research/gaia/gremlin/gremlin_core/src/storage.rs

```diff
@@ -411,7 +417,15 @@ impl LazyVertexDetails {
 }
 
 impl Details for LazyVertexDetails {
-    fn get_property(&self, key: &str) -> Option<BorrowObject> {
+    fn get_property(&self, key: &PropKey) -> Option<BorrowObject> {
+        let key = match key {
+            PropKey::Str(key) => key,
```

Comment:
I think, we should directly return None if it is prop_id as:
```
if let PropKey::Str(k) = key {
    ....
} else {
    info!(...);
    None
}
```

Reply:
Done